src/test/resources/feedback/workermessage1.txt

```diff
@@ -10,15 +10,12 @@ Rating:
 Feedback: Foobar1
 Quality: 1
 Rating: 1
-
 Rating:
 
 Feedback: Foobar2
 Quality: 2
 Rating: 2
-
 Additional feedback: awes0me Message
-
```

Comment:
Here should be a new line for better layout
[This](http://i.imgur.com/3du4fhN.png) looks a bit strange, I think.